libs/s25main/world/GameWorldBase.cpp

```diff
@@ -42,7 +42,10 @@ GameWorldBase::GameWorldBase(std::vector<GamePlayer> players, const GlobalGameSe
       gameSettings(gameSettings), em(em), gi(nullptr)
 {}
 
-GameWorldBase::~GameWorldBase() = default;
+GameWorldBase::~GameWorldBase()
+{
+    delete econHandler;
```

Comment:
When using `std::unique_ptr` this can be reverted. Generally: No new/delete wherever possible